stonesoup/updater/kalman.py

```diff
@@ -226,9 +227,8 @@ def predict_measurement(self, predicted_state, measurement_model=None,
         meas_cross_cov = self._measurement_cross_covariance(predicted_state, hh)
         innov_cov = self._innovation_covariance(meas_cross_cov, hh, measurement_model)
 
-        return GaussianMeasurementPrediction(pred_meas, innov_cov,
-                                             predicted_state.timestamp,
-                                             cross_covar=meas_cross_cov)
+        return MeasurementPrediction.from_state(
+            predicted_state, state_vector=pred_meas, covar=innov_cov, cross_covar=meas_cross_cov)
```

Comment:
The fact that the timestamp isn't passed explicitly fooled me a bit. But the tests pass, and if I follow the logic the timestamp is passed via the predicted state kwargs. Right?

Reply:
Yes it is, but could explicit include it to make it clearer.